lib/kitchen/driver/openstack.rb

```diff
@@ -85,12 +83,9 @@ def create(state)
         config[:disable_ssl_validation] && disable_ssl_validation
```

Comment:
I know this isn't part of your PR, but this syntax always bugs me, I don't know why.

I think using a guard clause instead reads nicer:

```ruby
disable_ssl_validation if config[:disable_ssl_validation]
```

It's a nitpick, so don't feel obligated to change it.